### DISCUSSION TITLE
环境变量名不宜为 `v2ray.vmess.aead.forced`，`v2ray_vmess_aead_forced` 为佳

### DISCUSSION THREAD

**Discussion by IceCodeNew** [v2fly/v2ray-core#811] [Ideas]
试着在 SSH 会话里敲一次 `v2ray.vmess.aead.forced=false` 就会明白了，变量名中间用连接符 `-` 或者小数点 `.` 都是不合适的。

**Comment by IceCodeNew** [comment]
![image](https://user-images.githubusercontent.com/32576256/111912222-ae175880-8aa3-11eb-8643-63600c13ef57.png)

**Comment by kslr** [comment]
这些大多是硬编码的，需要逐渐适配

**Comment by rurirei** [comment]
try this.
```cmd
"v2ray.vmess.aead.forced"=false
```

**Comment by AkinoKaede** [comment]
我印象中 V2RAY_VMESS_AEAD_FORCED 应该也是可以的…

**Reply by sixg0000d** [reply]
我印象里所有 x.x.x.x 在作环境变量使用时都可以用 X_X_X_X
https://github.com/v2fly/v2ray-core/blob/a21e4a7deb2d60b40ea4ebcedf6e22a56dcb4d04/common/platform/platform.go#L51-L53

https://www.v2fly.org/config/env.html
（另外吐槽下这个页面很难找到，不属于左侧任何章节，我是从历史记录翻进去的，都不记得入口在哪了）

如果是的话，把这段话改一下就行了
https://github.com/v2fly/v2ray-core/blob/a21e4a7deb2d60b40ea4ebcedf6e22a56dcb4d04/proxy/vmess/encoding/server.go#L200

**Reply by AkinoKaede** [reply]
那个页面的确挺难找，不过也过时了…
